### PR TITLE
HL-35

### PR DESCRIPTION
**Pull request by pondpiu**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-35)

- grey out and show winner on finished race

image

### CODE REVIEW

**Review by bitcoinbrisbane — APPROVED** [review]

File: src/pages/Dashboard/Dashboard_View.tsx [hunk 1]
```diff
@@ -204,23 +204,40 @@ const Table: React.FC<TableProps> = (props: TableProps) => {
...
-                                </Link>
-                              </>
-                            ) : (
-                              <Skeleton count={2} />
-                            )}
+                          <td>
+                            <div
+                              className={
+                                "px-3 py-4 whitespace-nowrap text-sm" +
+                                (race.status == "Paying"
```
Comment (brettvniekerk) [inline src/pages/Dashboard/Dashboard_View.tsx:211]: is there an enum for race status? seems prone to error to compare strings

Comment (brettvniekerk) [reply src/pages/Dashboard/Dashboard_View.tsx:211]: also could this not be a ternary in a string literal?

File: src/pages/Dashboard/Dashboard_View.tsx [hunk 2]
```diff
@@ -204,23 +204,40 @@ const Table: React.FC<TableProps> = (props: TableProps) => {
...
+                                  ? " bg-gray-400 hover:bg-gray-500"
+                                  : " hover:bg-gray-200")
+                              }
+                            >
+                              {race.name ? (
+                                <>
```
Comment (brettvniekerk) [inline src/pages/Dashboard/Dashboard_View.tsx:217]: there is no need for the fragment if there is only one parent tag - looks like the `Link` tag in this case

Comment (pondpiu) [reply src/pages/Dashboard/Dashboard_View.tsx:217]: nice catch

File: src/pages/Dashboard/Dashboard_View.tsx (hunk 1 above)
Comment (pondpiu) [reply src/pages/Dashboard/Dashboard_View.tsx:211]: hmmmm, i felt like it's harder to read
Screen Shot 2022-11-04 at 4 27 33 pm

Comment (brettvniekerk) [reply src/pages/Dashboard/Dashboard_View.tsx:211]: imo ternary in string literal is better than `string + string` - doesnt look much harder to read - not a critical change tho if you think the current approach is better

File: src/pages/Dashboard/Dashboard_View.tsx (hunk 2 above)
Comment (brettvniekerk) [reply src/pages/Dashboard/Dashboard_View.tsx]: :)

**Review by bitcoinbrisbane — APPROVED** [review]

**Review by brettvniekerk — APPROVED** [review]

**Review by brettvniekerk — APPROVED** [review]

**Pull request merged**